ArchiSteamFarm/ArchiWebHandler.cs

```diff
@@ -589,6 +589,113 @@ internal sealed class ArchiWebHandler : IDisposable {
 			}
 		}
 
+		[SuppressMessage("ReSharper", "FunctionComplexityOverflow")]
+		internal async Task<HashSet<Steam.Asset>> GetMyCommunityInventory(bool tradableOnly = false, string appId = null, string contextId = null, IReadOnlyCollection<uint> wantedRealAppIDs = null) {
```

Comment:
You shouldn't define another function that does exactly the same thing as current one. Instead, just make it accept custom `appID` and `contextID` that would default to current hardcoded settings.